TypeScript.YAML-tmLanguage

```diff
@@ -37,15 +37,15 @@ repository:
     end: (?=$|[;,])
     patterns:
     - include: '#ternary-operator'
-    - include: '#ternary-expression-type' 
+    - include: '#expression-type' 
 
   ternary-operator:
     begin: (\?)
     end: (:)
     patterns:
-    - include: '#ternary-expression-type' 
+    - include: '#expression-type' 
 
-  ternary-expression-type:
+  expression-type:
```

Comment:
Nopes, I just used it in case-clause because the expression in case clause had a recursive call for expression scope.